example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -731,6 +737,10 @@ describe('When there are granule differences and granule reconciliation is run',
     console.log(`update granule files back ${publishedGranuleId}`);
     await granuleModel.update({ granuleId: publishedGranuleId }, { files: JSON.parse(granuleBeforeUpdate.body).files });
 
+    // TODO there may be another execution to delete here
```

Comment:
Is this an errant comment, or is it something we're intending to clean up in another ticket?

Reply:
Yeah, this is one I'm not sure of. There are 2 specs with this comment. If there are unaccounted for execution errors these were going to be the first places to look. This may be moot with your previous comment though.